lib/AdapterRpc.php

```diff
@@ -59,8 +63,9 @@ public function __construct($configFileName = null)
         $this->rpcUrl = $conf->getString(self::RPC_URL);
         $this->rpcUser = $conf->getString(self::RPC_USER);
         $this->rpcPassword = $conf->getString(self::RPC_PASSWORD);
+        $this->rpcSerializer = $conf->getString(self::RPC_SERIALIZER);
```

Comment:
Please set default value to 'json':
$this->rpcSerializer = $conf->getString(self::RPC_SERIALIZER, 'json');

The file in config-templates is an only example configuration.

Reply:
Please fix errors from: https://www.codefactor.io/repository/github/cesnet/perun-simplesamlphp-module/pull/132 
and add this change into CHANGELOG.md too